src/Microsoft.Identity.Client/Platforms/Shared/Apple/BrokerConstants.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace Microsoft.Identity.Client.Platforms.Shared.Apple
```

Comment:
@bgavrilMS : is it related to this PR?

Reply:
I recommend reviewing commit by commit. Some of the commits are pure refactoring / moving of files, so not worth reviewing. The commit title is a good clue to understand this.